Lesson1 (Sequence containers)/Proskuchenko/lesson2task2/linkLevelNodes.cpp

```diff
@@ -0,0 +1,48 @@
+#include <iostream>
+#include <queue>
+#include <cmath>
+
+struct node {
+  int n = 1;
+  node* left = nullptr;
+  node* right = nullptr;
+  node* level = nullptr;
+};
+
+void linkLevelNodes(node* root) {
+  if (nullptr == root) {
+    return;
+  }
+  std::queue<node*> ptrNode;
+  ptrNode.push(root);
+  size_t NextLevelNodes = 0;
```

Comment:
it looks like useless action.
are you sure that you need variables at 18 and 19 lines?